include/canbus/isotp.h

```diff
@@ -103,7 +103,7 @@ struct isotp_msg_id {
 	/** Message identifier*/
 	union {
 		uint32_t std_id  : 11;
-		uint32_t ext_id  : 29;
+		uint32_t id  : 29;
 	};
```

Comment:
Any special reason to keep `std_id` here?

Reply:
Here we can keep the union for convenience. As long as we don't use another bit of the uint32_t, we can use the union.

Reply:
Renamed back to ext_id. This was a mistake.